src/profile-logic/convert-markers.js

```diff
@@ -73,12 +69,18 @@ export function upgradeGCMajorMarker(marker: Object) {
         timings.slices = timings.slices.length;
       }
 
-      timings.phase_times = _upgradePhaseTimes(timings.totals);
+      if (upgrade_processed) {
+        timings.phase_times = convertPhaseTimes(timings.totals);
+      } else {
+        timings.phase_times = timings.totals;
+      }
       delete timings.totals;
 
-      timings.mmu_20ms /= 100;
-      timings.mmu_50ms /= 100;
-      timings.allocated *= 1024 * 1024;
+      if (upgrade_processed) {
+        timings.mmu_20ms /= 100;
+        timings.mmu_50ms /= 100;
```

Comment:
Yeah, that's why it's only done if upgrade_processed is true.  For upgrading the processed profile.  My goal is that the processed profile is how I think this data _should_ have been all allong.  But we don't want gecko to use this format because that would break telemety.